tests/test_github_issue_14/pythonTest.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+import sys
+import os
+import local_script
+
+# Ensure that the __real_file__ path ends in .snakemake
+dname = os.path.dirname(__real_file__)
+print(dname)
+if not dname.endswith(".snakemake/scripts"):
```

Comment:
This will not work on Windows where path separators are `\`

Reply:
I didn't write that (note that it's from another test) but I'm happy to change it.

Reply:
See https://github.com/snakemake/snakemake/blob/master/tests/test_issue916/pythonTest.py#L9